metadata-app/src/components/RequiredMetadata/index.js

```diff
@@ -27,11 +27,11 @@ const RequiredMetadata = (props) => {
       // if it comes from organization page then pre-select that specific organization
       const urlParams = new URLSearchParams(window.location.search);
       const orgId = urlParams.get('group');
-      if (orgId) values.owner_org = orgId;
+      if (orgId) setFieldValue('owner_org', orgId);
```

Comment:
I'm surprised our linter didn't catch this. I think one-line if statements should have braces. At least to be consistent with the if statement below.

Reply:
@adborden I'll add the curly braces, agree. 

I just checked the ESlint config we don't have such rule. I just tired to add it, but it seems we omitted the curly braces in other files too. So probably would be better to create a separate PR for this new linting rule.